GDJS/Runtime/layer.js

```diff
@@ -396,21 +407,44 @@ gdjs.Layer.prototype._setEffectsDefaultParameters = function() {
  * time will be slower if time scale is < 1, faster if > 1.
  * @param {number} timeScale The new time scale (must be positive).
  */
-gdjs.Layer.prototype.setTimeScale = function(timeScale) {
+gdjs.Layer.prototype.setTimeScale = function (timeScale) {
   if (timeScale >= 0) this._timeScale = timeScale;
 };
 
 /**
  * Get the time scale for the objects on the layer.
  */
-gdjs.Layer.prototype.getTimeScale = function() {
+gdjs.Layer.prototype.getTimeScale = function () {
   return this._timeScale;
 };
 
 /**
  * Return the time elapsed since the last frame,
  * in milliseconds, for objects on the layer.
  */
-gdjs.Layer.prototype.getElapsedTime = function() {
+gdjs.Layer.prototype.getElapsedTime = function () {
   return this._runtimeScene.getTimeManager().getElapsedTime() * this._timeScale;
 };
+
+/**
+ * Set the position of layer's camera to follow
+ * base layer's camera.
+ */
+gdjs.Layer.prototype.followBaseLayer = function () {
+  this.setCameraX(this._runtimeScene.getLayer('').getCameraX());
+  this.setCameraY(this._runtimeScene.getLayer('').getCameraY());
+};
+
+/**
+ * @return {number[]} the clear color of layer in the range of [0, 1].
+ */
+gdjs.Layer.prototype.getClearColor = function () {
+  return this._clearColor;
+};
+
+/**
+ * @return {boolean} true if it is a lighting layer, false otherwise.
+ */
+gdjs.Layer.prototype.isLightingLayer = function () {
+  return this._isLightingLayer;
+}
```

Comment:
nit: configure your editor to always add a line break at the end of the file ("posix" good practice)